src/api/spec/models/cloud/azure/configuration_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+# WARNING: Some tests require real backend answers, so make sure you uncomment
+# this line and start a test backend.
+# CONFIG['global_write_through'] = true
+
+RSpec.describe Cloud::Azure::Configuration, type: :model, vcr: true do
+  describe 'data encryption' do
+    let(:config) { create(:azure_configuration, application_id: 'Hey OBS!', application_key: 'Hey OBS?') }
+    let(:secret_key) { OpenSSL::PKey::RSA.new(File.read(Rails.root.join('spec', 'support', 'files', 'cloudupload_secret_key_tests.pem'))) }
```

Comment:
I don't understand why the model uses this key? I would expect that we need to mock a backend call?
Also I don't like that we create always a new key, why is this necessary as we already commit the key to git?

Reply:
@ChrisBr 

> I don't understand why the model uses this key? I would expect that we need to mock a backend call?

Done. The request done by the backend call is now stubbed.

> Also I don't like that we create always a new key, why is this necessary as we already commit the key to git?

The secret_key is in a let clause. And now the public_key is also in a let clause. So reading the keys will be executed one time.